content/posts/0006-build-plan-buildpack.md

```diff
@@ -0,0 +1,93 @@
+---
+title: What is the Build Plan Buildpack and why should I care?
+date: "2021-02-09"
+slug: build-plan-buildpack-exploration
+author: feckhardt
+---
+The [Build Plan buildpack](https://github.com/paketo-community/build-plan)
+gives you very precise control over what buildpacks run during a build and what
+dependencies end up in the final image. Let's talk about how it works and
+why/when you would utilize this kind of functionality.
+
+### Brief summary of the Build Plan
+
+The Build Plan is a part of the Cloud Native Buildpacks (CNB) Specification
+that is the format for how buildpacks communicate during the detect phase of
+the lifecycle. Buildpacks essentially write out a Build Plan that outlies the
+dependency that the buildpack will either provide or require during its build
+phase. Buildpacks may both require and provide any number of dependencies. For
+detection to pass a buildpacks must have all of the dependencies it requires,
+provided by a buildpack that comes before it in the build order and it needs to
+have all of the dependencies that it provides, required by buildpacks that come
+later in the build order. If either of these conditions is not satisfied
+detection will fail for that buildpack. A more in-depth breakdown of the Build
+Plan can be found in the [CNB
+Spec](https://github.com/buildpacks/spec/blob/main/buildpack.md).
```

Comment:
I wonder if this can be explained a bit more clearly? It's a pretty central concept to the blog post and this wording is kind of confusing. Maybe a diagram could help, if you're into that